sea-orm-migration/src/migrator.rs

```diff
@@ -289,6 +289,70 @@ pub trait MigratorTrait: Send {
 
         Ok(())
     }
+
+    /// Apply or Rollback migrations to version
+    async fn change_to_version(db: &DbConn, version: &str) -> Result<(), DbErr> {
+        Self::up_to_version(db, version).await?;
+        Self::down_to_version(db, version).await
+    }
+
+    /// Apply migrations to version
+    async fn up_to_version(db: &DbConn, version: &str) -> Result<(), DbErr> {
+        let steps = Self::get_steps(db, MigrationStatus::Pending, version).await?;
+        if steps > 0 {
+            Self::up(db, Some(steps)).await
+        } else {
+            Ok(())
+        }
+    }
+
+    /// Rollback migrations to version
+    async fn down_to_version(db: &DbConn, version: &str) -> Result<(), DbErr> {
+        let steps = Self::get_steps(db, MigrationStatus::Applied, version).await?;
+        if steps > 1 {
+            Self::down(db, Some(steps - 1)).await
+        } else {
+            Ok(())
+        }
+    }
+
+    async fn get_steps(db: &DbConn, status: MigrationStatus, version: &str) -> Result<u32, DbErr> {
+        let mut index = 0;
+        let mut matched = false;
+        let migrations = Self::get_migration_with_status(db).await?;
+        match status {
+            MigrationStatus::Pending => {
+                for migration in migrations {
+                    if migration.status == MigrationStatus::Pending {
+                        index += 1;
+                    }
```

Comment:
I think we should use `get_pending_migrations` method to get a list of pending migration. To keep the behaviour consistent with the `up` method.

Reply:
If this is the case ( [#755](https://github.com/SeaQL/sea-orm/pull/755) ）, Migration V1: Applied，Migration V2: Pending，Migration V3: Applied, Migration V4: Applied,  get_pending_migrations  method has not V3